src/editor/ContentSrc.jsx

```diff
@@ -0,0 +1,59 @@
+var React = require('react');
+var CM = require('codemirror');
+
+var CodeMirror = React.createClass({
+	propTypes: {
+		options: React.PropTypes.object,
+		value: React.PropTypes.string
```

Comment:
Would love to see getValue, setValue, blur, onBlur functions all defined here as public interface for the editor to call.